.github/workflows/ci.yaml

```diff
@@ -29,6 +29,10 @@ jobs:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11"]
         pydantic-version: ["1.10", "2.0"]
+        sqla-version: ["1.4", "2"]
+        exclude:
```

Comment:
Why is this needed?

Reply:
This excludes extra runs for when 1.4 and 1.10 is installed. This feels alright to exclude given implementation of factories being separate and avoiding extra potential CI time as may start hitting limit of concurrent runs.